chia/util/config.py

```diff
@@ -325,3 +327,18 @@ def load_defaults_for_missing_services(config: Dict[str, Any], config_name: str)
                     defaulted[service]["selected_network"] = "".join(to_be_referenced)
 
     return defaulted
+
+
+PEER_INFO_MAPPING: Dict[NodeType, str] = {
+    NodeType.FULL_NODE: "full_node_peer",
+    NodeType.FARMER: "farmer_peer",
+}
+
+
+def get_unresolved_peer_infos(service_config: Dict[str, Any], peer_type: NodeType) -> Set[UnresolvedPeerInfo]:
+    peer_info_key = PEER_INFO_MAPPING[peer_type]
+    peer_infos = service_config.get(f"{peer_info_key}s", [])
+    peer_info = service_config.get(peer_info_key)
+    all_peer_infos_raw = [peer_info, *peer_infos] if peer_info is not None else peer_infos
```

Comment:
These ternaries end up keeping coverage from checking that we cover both cases so I am kind of trying to avoid them.  Though, I haven't gone quite so far yet as to add a linter check to ban them.

What do you think of something like this?

```suggestion
    peer_infos: List[Dict[str, Any]] = service_config.get(f"{peer_info_key}s", [])
    peer_info: Optional[Dict[str, Any]] = service_config.get(peer_info_key)
    if peer_info is not None:
        peer_infos.append(peer_info)
```